microsoft-teams/1.0/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,60 @@
+= Microsoft Teams Connector 1.0 - Mule 4
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for Microsoft Teams (Microsoft Teams Connector) gives you access to the Microsoft Teams APIs. This connector exposes the most commonly used operations provided by the Microsoft Teams API. 
```

Comment:
Is it ONE Microsoft Teams API or "the Microsoft Teams APIs"? Clarify and make consistent.